api/ssi-bridge/src/services/ssi-service.ts

```diff
@@ -203,31 +213,40 @@ export class SsiService {
 		}
 	}
 
-	async generateIdentity(): Promise<{ account: Identity.Account; doc: Identity.Document; key: Identity.KeyPair }> {
+	async generateIdentity(): Promise<{
+		doc: Identity.Document;
+		key: Identity.KeyPair;
+		encryption: Identity.KeyPair;
+	}> {
 		try {
-			const builder = this.getAccountBuilder();
-			const keyPair = new KeyPair(KeyType.Ed25519);
-			const account = await builder.createIdentity({ privateKey: keyPair.private() });
+			const signKeyPair = new KeyPair(KeyType.Ed25519);
+			const document = new Document(signKeyPair, this.getConfig(false).network.name());
+
+			// Add encryption keys and capabilities to Identity
+			const encryptionKeyPair = new KeyPair(KeyType.X25519);
+			const encryptionMethod = new Identity.VerificationMethod(
+				document.id(),
+				encryptionKeyPair.type(),
+				encryptionKeyPair.public(),
+				'kex-0'
+			);
+			document.insertMethod(encryptionMethod, MethodScope.KeyAgreement());
+
+			document.signSelf(signKeyPair, document.defaultSigningMethod().id());
+			const client = await this.getIdentityClient(false);
+			await client.publishDocument(document);
 
 			return {
-				account,
-				doc: account.document(),
-				key: keyPair
+				doc: document,
+				key: signKeyPair,
```

Comment:
can you please rename `key` to `signingKeys` or `signatureKeys` and encryption to `encryptionKeys` ?